### PR TITLE
fix: MD012/no-multiple-blanks

### PR DESCRIPTION
**Pull request by nschonni**
Remove multiple consecutive blanks.
Find:\n\n\n
Replace:\n\n
Filter:.md

### CODE REVIEW

**Review by 3imed-jaberi — APPROVED** [review]

File: locale/en/blog/npm/npm-1-0-link.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ I highly doubt that a native Windows node will ever have comparable symbolic lin
 
 When there is a native windows port of Node, if that native windows port has \`fs.symlink\` and \`fs.readlink\` support that is exactly identical to the way that they work on Unix, then this should work fine.
 
-<h2 id="aside_credit_where_credit8217s_due">Aside: Credit where Credit&#8217;s Due</h2>
+But I wouldn't hold my breath. Any bugs about this not working on a native Windows system (ie, not Cygwin) will most likely be closed with `wontfix`.
```
Comment (nschonni) [inline locale/en/blog/npm/npm-1-0-link.md]: Don't think the conflict resolution was right here. I'll rebase this to fix it properly

File: locale/en/blog/release/v0.8.0.md [hunk 2]
```diff
@@ -96,14 +96,12 @@ Requests per second:    3317.24 [#/sec] (mean)
 # 0.8.0
 Requests per second:    3795.34 [#/sec] (mean) +14.41%
 
-
```
Comment (XhmikosR) [inline locale/en/blog/release/v0.8.0.md:99]: This changes line in `pre`s too.

**Review by ghost — APPROVED** [review]
Just a confirm：locale/en/blog/announcements/interactive-2016-ams.md, it seems that we should keep the last line as a newline (L84), but this change will remove it? If we don't have such problems, this LGTM for me.

File: locale/en/blog/release/v0.8.0.md (hunk 2 above)
Comment (nschonni) [reply locale/en/blog/release/v0.8.0.md:99]: Good catch!
Looking at the other blocks in this file, the double line in the other calls isn't consistent, so i don't think it matters in this case

**Comment by nschonni** [comment]
@MaledongGit in that file, it had 2 trailing space instead of the single trailing line before the EOF

PS: If both trailing lines were removed, then MD047 would have gotten triggered

**Pull request merged**